client/src/modules/employees/registration/employees.html

```diff
@@ -0,0 +1,513 @@
+                        <div ng-messages-include="modules/templates/dates.messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Gender -->
+                  <div id="gender" class="form-group"
+                    ng-class="{'has-error' : DetailForm.genderOptions.$invalid && EmployeeRegistrationForm.$submitted}">
+                    <label class="col-md-3 control-label" translate>FORM.LABELS.GENDER</label>
+                    <div class="col-md-9">
+                      <label class="radio-inline">
+                        <input type="radio" name="genderOptions" id="male" value="M" ng-model="EmployeeCtrl.employee.sex" required>
+                        <span translate>FORM.LABELS.MALE</span>
+                      </label>
+
+                      <label class="radio-inline">
+                        <input type="radio" name="genderOptions" id="female" value="F" ng-model="EmployeeCtrl.employee.sex" required>
+                        <span translate>FORM.LABELS.FEMALE</span>
+                      </label>
+
+                      <div class="help-block" ng-messages="DetailForm.genderOptions.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee code  -->
+                  <div
+                    class="form-group"
+                    ng-class="{'has-error' : DetailForm.code.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label" translate>FORM.LABELS.CODE</label>
+                    <div class="col-md-9">
+                      <input
+                        class="form-control"
+                        ng-maxlength="EmployeeCtrl.length20"
+                        name="code"
+                        ng-model="EmployeeCtrl.employee.code"
+                        autocomplete="off"
+                        required>
+                      <div class="help-block" ng-messages="DetailForm.code.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee grade-->
+                  <div class="form-group"
+                    ng-class="{'has-error' : DetailForm.grade.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <!-- Warning for incomplete data - the page will not be able to continue -->
+                    <uib-alert class="text-danger" type="danger" ng-if="EmployeeCtrl.grades.length === 0">
```

Comment:
Yes but I think not for this PR I think. An other PR will follow for editing and other functionalities and I can implement it over there.
What do you think?